new smart contracts/contracts/CollectionFactory.sol

```diff
@@ -77,7 +77,7 @@ contract CollectionFactory is Ownable {
                         collectionMerkleRoot,
                         block.chainid,
                         msg.sender,
-                        nonces[msg.sender]
+                        nonces[msg.sender] + 1
```

Comment:
what is the nonce tracking now? It made sense before but does it still make sense ?

Reply:
it's tracking the DetachNonce in Clamor

Reply:
yeah but why do we need that still? can't rem if there is a reason anymore (fragnova side)

Reply:
I'm not sure what you mean. So you're suggesting that on Fragnova's side we should just read the CollectionFactory.sol's `nonces` state variable?

Reply:
> I'm not sure what you mean. So you're suggesting that on Fragnova's side we should just read CollectionFactory.sol's `nonces` state variable ? I guess that could also work 🤔

But maybe using an Ethereum RPC Node Provider can be unreliable 🤔?

Reply:
no, I'm thinking we can remove and simplify

Reply:
If we remove it, then the same user can replay `CollectionFactory.attachCollection()` unlimited times

Reply:
> I'm not sure what you mean. So you're suggesting that on Fragnova's side we should just read CollectionFactory.sol's `nonces` state variable ? I guess that could also work 🤔

Actually nvm, I don't think this is possible also